.github/workflows/call-windows-unit-tests.yaml

```diff
@@ -0,0 +1,162 @@
+          echo "date=$(/bin/date -u "+%Y%m%d")" >> $GITHUB_OUTPUT
+        shell: bash
+
+      - name: Restore cached packages of vcpkg
+        id: cache-unit-test-vcpkg-sources
+        uses: actions/cache/restore@v4
+        with:
+          path: |
+            C:\vcpkg\packages
+          key: ${{ runner.os }}-${{ matrix.config.arch }}-wintest-vcpkg-${{ steps.get-date.outputs.date }}
+          restore-keys: |
+            ${{ runner.os }}-${{ matrix.config.arch }}-wintest-vcpkg-
+          enableCrossOsArchive: false
+
+      - name: Build openssl with vcpkg
+        run: |
+          C:\vcpkg\vcpkg install --recurse openssl --triplet ${{ matrix.config.vcpkg_triplet }}
+        shell: cmd
+
+      - name: Build libyaml with vcpkg
+        run: |
+          C:\vcpkg\vcpkg install --recurse libyaml --triplet ${{ matrix.config.vcpkg_triplet }}
+        shell: cmd
+
+      - name: Save packages of vcpkg
+        id: save-vcpkg-sources
+        uses: actions/cache/save@v4
+        with:
+          path: |
+            C:\vcpkg\packages
+          key: ${{ steps.cache-unit-test-vcpkg-sources.outputs.cache-primary-key }}
+          enableCrossOsArchive: false
+
+      - name: Build unit-test for Fluent Bit packages (only for x86 and x64)
+        run: |
+          cmake -G "NMake Makefiles" `
+            -D FLB_TESTS_INTERNAL=On `
+            -D FLB_NIGHTLY_BUILD='${{ inputs.unstable }}' `
+            -D OPENSSL_ROOT_DIR='${{ matrix.config.openssl_dir }}' `
+            ${{ matrix.config.cmake_additional_opt }} `
+            -D FLB_LIBYAML_DIR=C:\vcpkg\packages\libyaml_${{ matrix.config.vcpkg_triplet }} `
+            -D FLB_WITHOUT_flb-rt-out_elasticsearch=On `
+            -D FLB_WITHOUT_flb-rt-out_td=On `
+            -D FLB_WITHOUT_flb-rt-out_forward=On `
+            -D FLB_WITHOUT_flb-rt-in_disk=On `
+            -D FLB_WITHOUT_flb-rt-in_proc=On `
+            -D FLB_WITHOUT_flb-it-parser=On `
+            -D FLB_WITHOUT_flb-it-unit_sizes=On `
+            -D FLB_WITHOUT_flb-it-network=On `
+            -D FLB_WITHOUT_flb-it-pack=On `
+            -D FLB_WITHOUT_flb-it-signv4=On `
+            -D FLB_WITHOUT_flb-it-aws_credentials=On `
+            -D FLB_WITHOUT_flb-it-aws_credentials_ec2=On `
+            -D FLB_WITHOUT_flb-it-aws_credentials_http=On `
+            -D FLB_WITHOUT_flb-it-aws_credentials_profile=On `
+            -D FLB_WITHOUT_flb-it-aws_credentials_sts=On `
+            -D FLB_WITHOUT_flb-it-aws_util=On `
+            -D FLB_WITHOUT_flb-it-input_chunk=On `
+            ../
+          cmake --build .
+        shell: pwsh
+        working-directory: build
+
+      - name: Upload unit test binaries
+        uses: actions/upload-artifact@v4
+        with:
+          name: windows-unit-tests-${{ matrix.config.arch }}
+          path: |
+            build/**/*.exe
+          if-no-files-found: error
+
+      - name: Display dependencies w/ dumpbin
+        run: |
+          dumpbin /dependents .\bin\fluent-bit.exe
```

Comment:
I think this should be `build` and maybe the unit test binaries?
It's failing at the moment.

Reply:
```suggestion
          dumpbin /dependents .build\bin\fluent-bit.exe
```